src/components/Modals/CreateMultipleItemsModal/CreateMultipleItemsModal.tsx

```diff
@@ -5,14 +5,23 @@ import Dropzone, { DropzoneState } from 'react-dropzone'
 import { env } from 'decentraland-commons'
 import { Button, Icon, Message, ModalNavigation, Progress, Table } from 'decentraland-ui'
 import Modal from 'decentraland-dapps/dist/containers/Modal'
+import { omit } from 'decentraland-commons/dist/utils'
 import { T, t } from 'decentraland-dapps/dist/modules/translation/utils'
 import { MultipleItemsSaveState } from 'modules/ui/createMultipleItems/reducer'
 import { BuiltFile, IMAGE_PATH } from 'modules/item/types'
 import { generateCatalystImage } from 'modules/item/utils'
 import ItemImport from 'components/ItemImport'
 import { InfoIcon } from 'components/InfoIcon'
 import { buildThirdPartyURN, DecodedURN, decodeURN, URNType } from 'lib/urn'
-import { ImportedFile, ImportedFileType, ItemCreationView, Props, RejectedFile, State } from './CreateMultipleItemsModal.types'
+import {
+  CreateOrEditMultipleItemsModalType,
+  ImportedFile,
+  ImportedFileType,
+  ItemCreationView,
+  Props,
+  RejectedFile,
+  State
+} from './CreateMultipleItemsModal.types'
```

Comment:
Should we rename the modal to `CreateAndEditMultipleItemsModal`?

Reply:
sure! done!